msteams-platform/whats-new.md

```diff
@@ -17,6 +17,7 @@ Discover Microsoft Teams platform features that are generally available (GA) and
 
 | Date | Update | Find here |
 | --- | --- | --- |
+|03/16/2022| Added information on debug process| Tools and SDKs > Teams Toolkit for Visual Studio Code > [Debug your Teams app using Teams Toolkit](toolkit/debug-local.md) > [Debug background process](toolkit/debug-background-process.md)|
```

Comment:
"Debug background process" page is in the same level as "Debug your Teams app using Teams Toolkit" page. So I think it is better to use the format like:
![image](https://user-images.githubusercontent.com/37978464/158819105-503ca139-1507-4020-97e9-047898031657.png)